dm/proto/dmworker.proto

```diff
@@ -116,6 +116,8 @@ message DumpStatus {
     double finishedBytes = 3;
     double finishedRows = 4;
     double estimateTotalRows = 5;
+    int64 bps = 6;
+    string progress = 7;
```

Comment:
for `finished-unit-status`, it's using this struct too, maybe add `duration` in https://github.com/pingcap/tiflow/pull/7507/ here

Reply:
yes, duration is in https://github.com/pingcap/tiflow/pull/7507, what's your problem?

Reply:
duration should put into this struct too, so we can get it when the unit has finished

Reply:
Ref this [comment](https://github.com/pingcap/tiflow/issues/7343#issuecomment-1276969371),  we need manage the unit's `duration` in engine layer.

Reply:
finished unit need duration too. we can discuss in that pr